src/Adapters/EventSource.Netstandard13/EventSourceTelemetryModule.cs

```diff
@@ -154,9 +154,9 @@ protected override void OnEventSourceCreated(EventSource eventSource)
         private void EnableAsNecessary(EventSource eventSource)
         {
             // Special case: enable TPL activity flow for better tracing of nested activities.
-            if (eventSource.Guid == EventSourceTelemetryModule.TplEventSourceGuid)
+            if (eventSource.Guid == TplActivities.TplEventSourceGuid)
             {
-                this.EnableEvents(eventSource, EventLevel.LogAlways, (EventKeywords)EventSourceTelemetryModule.TaskFlowActivityIds);
+                this.EnableEvents(eventSource, EventLevel.LogAlways, (EventKeywords) TplActivities.TaskFlowActivityIdsKeyword);
```

Comment:
@vancem @SergeyKanzhelev Event if net46 has a problem, this telemetry module targets netstandard 1.3, so it is useful for .NET Core apps too